src/OpenTelemetry.Api/Trace/TelemetrySpan.cs

```diff
@@ -303,6 +304,69 @@ public TelemetrySpan AddBaggage(string key, string value)
             return this;
         }
 
+        /// <summary>
+        /// Record Exception.
+        /// </summary>
+        /// <param name="ex">Exception to be recorded.</param>
+        /// <returns>The <see cref="TelemetrySpan"/> instance for chaining.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public TelemetrySpan RecordException(Exception ex)
+        {
+            if (ex == null)
+            {
+                return this;
+            }
```

Comment:
you can call the other overload from this method instead of duplicating logic here.

Reply:
done!